src/org/opendatakit/aggregate/parser/BaseFormParserForJavaRosa.java

```diff
@@ -672,7 +672,7 @@ protected BaseFormParserForJavaRosa(String existingXml, String existingTitle, bo
       }
     }
     // clean illegal characters from title
-    title = formTitle.replace(BasicConsts.FORWARDSLASH, BasicConsts.EMPTY_STRING);
+    title = formTitle.replaceAll(BasicConsts.INVALID_CHARS, BasicConsts.EMPTY_STRING);
```

Comment:
There's already effort to sanitize the title, so I'm just updating the implementation which will replace more chars from the title. The title is what being used to create the folder name and the file name for the form.